tar/chunks/chunks_delete.c

```diff
@@ -163,21 +163,21 @@ chunks_delete_printstats(FILE * stream, CHUNKS_D * C, const char * name,
 		name = "This archive";
 
 	/* Print header. */
-	if (chunks_stats_printheader(stream, csv))
+	if (chunks_stats_printheader(stream, csv, 0))
```

Comment:
If we wanted to allow `tarsnap -d --null-output -f archivename`, then we'd hook it in here.

A small part of me thinks "hey, why not allow `--null-output` everywhere!".  But a larger part of me points out that it's not needed for the GUI, and I struggle to think of any scenario where `-d --null-output` would really be useful.